dashboard/test/ui/features/applab/dataBlocks.feature

```diff
@@ -1,13 +1,12 @@
 @no_ie9
 @dashboard_db_access
+@as_student
 Feature: App Lab Data Blocks
```

Comment:
I know these tags can be placed on the feature or the scenario. Can they also be placed on the background? For tags like this that result in specific actions (rather than toggling whether the test is run or not) background feels like a better place for tags that apply to all scenarios.